.github/workflows/ci.yml

```diff
@@ -44,23 +46,30 @@ jobs:
             - name: Install Composer dependencies
               run: composer install --no-ansi --no-interaction --no-suggest --no-progress --prefer-dist --optimize-autoloader --ignore-platform-reqs
 
-            - name: Prepare the database
+            - name: Prepare Database
               run: touch database/database.sqlite
 
-            - name: Prepare the application
+            - name: Prepare Application
               run: cp .env.example .env && php artisan key:generate
 
-            - name: Run PHPStan
-              run: composer analyse
+            - name: Run Rector
+              run: composer refactor
 
-    format:
+            - name: Run php-cs-fixer
+              run: composer format
```

Comment:
@faustbrian Is it ok for you running rector, then php-cs-fixer in the same job? or do you prefer having them separated in different jobs?
Like: `run rector, commit changes` -> `run php-cs-fixer, commit changes` -> `run php-stan` if all good -> `run tests` and `run e2e` concurrently